benchmarks/tpcc/src/main/java/com/google/cloud/pgadapter/tpcc/JavaClientBenchmarkRunner.java

```diff
@@ -0,0 +1,257 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.cloud.pgadapter.tpcc;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.pgadapter.tpcc.config.PGAdapterConfiguration;
+import com.google.cloud.pgadapter.tpcc.config.SpannerConfiguration;
+import com.google.cloud.pgadapter.tpcc.config.TpccConfiguration;
+import com.google.cloud.spanner.AbortedException;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.SessionPoolOptions;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.TransactionManager;
+import com.google.cloud.spanner.Value;
+import com.google.common.base.Stopwatch;
+import com.google.common.base.Strings;
+import com.google.common.util.concurrent.Uninterruptibles;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class JavaClientBenchmarkRunner extends AbstractBenchmarkRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(JavaClientBenchmarkRunner.class);
+  private final ThreadLocal<Spanner> spannerThreadLocal = new ThreadLocal<>();
```

Comment:
`Spanner` instances are intended a long-lived shared objects. So we should not create a separate `Spanner` instance for each thread.
The same also applies to `DatabaseClient`, but that happens automatically if you use a single `Spanner` instance. Multiple calls to `Spanner.getDatabaseClient(DatabaseId)` will return the same `DatabaseClient` instance.

Reply:
Updated `spanner` to be a shared object and removed `DatabaseClient`.